Dockerfile

```diff
@@ -74,9 +68,19 @@ RUN set -ex; \
     mkdir -p /var/lib/uwsgi; \
     mv dogstatsd_plugin.so /var/lib/uwsgi/; \
     uwsgi --need-plugin=/var/lib/uwsgi/dogstatsd --help > /dev/null; \
-    snuba --help; \
     \
-    apt-get purge -y --auto-remove $buildDeps
+    apt-get purge -y --auto-remove $buildDeps; \
+    rm -rf /var/lib/apt/lists/*;
+
+# Layer cache is pretty much invalidated here all the time,
+# so try not to do anything heavy beyond here.
```

Comment:
This gets invalidated at every commit in snuba, right?

Reply:
Yes.